docs/js/contribution.md

```diff
@@ -0,0 +1,173 @@
+---
+id: contribution-model-sap-cloud-sdk-for-javascript-typescript
+title: Contribution model
+sidebar_label: Contribution model
+description:
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+---
+
+## Introduction
+
+SAP Cloud SDK for JavaScript welcomes contributions that bring value to SDK users by simplifying and/or improving cloud application development within SAP ecosystem. The contributed feature or API library becomes a part of SDK and is released under SAP Cloud SDK umbrella into `@sap-cloud-sdk` to the [npmjs.com](https://www.npmjs.com/).
+
+Most modules of the SAP Cloud SDK for JavaScript are Open Source released under Apache 2.0 license. It's preferred, but not mandatory that contribution happens under same conditions.
+
+## Roles
+
+    - **SDK team** - development team and Product Owner responsible for maintaining, developing and releasing SAP Cloud SDK for JavaScript.
+    - **Contribution team** - development team responsible for developing and contributing a feature or an API library
+
+### Stakeholders and contact points
+
+    - **SDK Team** assigns two main stakeholders as contact points for the contribution process.
+      - **Prodact owner**  for organisational topics, priorities and practicalities
+      - **Developer** for engineering onboarding, alignment and consulting
+    - **Contribution team** assigns at least one contact point with engineering background for regular collaboration and syncs.
+
+Other stakeholders and/or contact points could be assigned if deemed necessary.
+
+## Process outline
+
+Below are generic steps for a successful contribution. The process can be adopted for the needs of specific contribution.
+
+    - Initial alignment and approvals
+    - Plan conribution collaboration
```

Comment:
typo

Reply:
I would add a substructure here since it has many points. 
### Phase I - Onboarding
- All initial preparation task

### Phase II - Get a Beta Out
- All development points
- High frequency of syncs
- PR review etc.
- Ends with the beta release

### Phase III stabalizing and GA
- Iterate with lower consulting frequency
- Plan publish
- Customer supports onboarding
- End with G

Reply:
That would be a fabulous update, but I think the current version is a good start so that I'm not blocked taking it to perfection. Would it be an option to merge it and then everyone can add parts where they have most knowledge or best idea on organizing it?

I can't be the only editor or we'll be stuck in reviews forever.